src/main/resources/index.html

```diff
@@ -70,6 +70,20 @@
       padding: 1rem;
       vertical-align: middle;
     }
+    #directions-hint {
+      color: #666;
+      font-size: .8em;
+      font-weight: normal;
+      position: fixed;
+      left: 50%;
+      transform: translateX(-50%);
+      top: 2.5rem;
```

Comment:
Place the `directions-hint` box in a fixed position near the bottom of the interface rather than the top.